go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/eoscanada/eos-go
```

Comment:
Please don't change this one, it will mess up other repos that using `github.com/eoscanada/eos-go`
In other repos just replace `github.com/eoscanada/eos-go` with `github.com/ultraio/eos-go` instead